tests/test_files.py

```diff
@@ -469,6 +470,18 @@ def test_manalyze(self, mock_get=None, mock_post=None):
         ).start()
         self.assertEqual(report.get("success", False), True)
 
+    def test_mwdb_scan(self, mock_get=None, mock_post=None):
```

Comment:
need to apply appropriate mock decorators here. See line 459, 460 for suitable example.